packages/storage/test/testshared.ts

```diff
@@ -16,9 +16,9 @@
  */
 import { expect } from 'chai';
 import { FirebaseApp } from '@firebase/app-types';
+import { contains, forEach } from '@firebase/util';
```

Comment:
I would also consider just using the built-in primitives.

Reply:
Done in next PR.